### DISCUSSION TITLE
keyball44の右側の右上のキーだけ反応しません

### DISCUSSION THREAD

**Discussion by Hama-Taka** [Yowkees/keyball#389] [Q&A]
右側の右上のキーのみ反応しません。（添付の画像では11の部分）
その他の反応しないキーはキーソケットのはんだづけを調整して反応するようになったのですが、
この右上のキーだけははんだを調整しても改善しません。
もしかしたらダイオードのはんだづけにも問題があるかもしれないので、
該当キーのダイオードの位置を教えていただけないでしょうか。
よろしくお願いします。
![image0](https://github.com/Yowkees/keyball/assets/147972769/951df894-b59b-4e2b-84a6-f79d83e8a376)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

問題のキーに該当するダイオードは下写真の青丸で示したものになります。
確かに半田が球状なので、テンプラしている可能性があります。半田が基板のパッドに馴染むように修正してみてください。

以上、お手数ですがご対応お願いいたします。

![KB44_rtkeyfix](https://github.com/Yowkees/keyball/assets/84209772/a528e204-29fa-46e9-825d-b61154b1d827)

**Reply by Hama-Taka** [reply]
早速の返信ありがとうございます。
なんとか、すべてのキーが無事反応するようになりました。
これから、キーマップのカスタマイズなど使い勝手がよくなるように頑張っていきたいと思います。
また、何かあれば質問させていただくかもしれませんが、何卒よろしくお願いします。

**Comment by Yowkees** [comment]
無事にキーが反応するようになったようで良かったです。安心しました。
ぜひ末永くご愛用いただければ幸いです。
今後ともどうぞよろしくお願いいたします。
